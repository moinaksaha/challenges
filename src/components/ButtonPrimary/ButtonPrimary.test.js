import React from 'react'
import renderer from 'react-test-renderer'
import Buttonprimary from './index.js'

it('Buttonprimary: default', () => {
  const component = renderer.create(<Buttonprimary />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})