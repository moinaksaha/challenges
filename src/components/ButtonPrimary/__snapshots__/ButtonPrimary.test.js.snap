// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <ButtonPrimary /> Snapshot: Enzyme 1`] = `
<styled.div>
  Button Test
</styled.div>
`;
